scripts/bug_classifier.py

```diff
@@ -69,10 +69,12 @@ def classify_bugs(model_name, classifier, bug_id):
         else:
             probas = model.classify(bug, probabilities=True, importances=False)
 
-        if np.argmax(probas) == 1:
-            print(f"Positive! {probas}")
+        probability = probas[0]
```

Comment:
You can calculate np.argmax(probability) here, so you don't have to repeat it twice in the if-else